src/components/core/Dropdown/SettingsDropdown.tsx

```diff
@@ -14,7 +14,8 @@ type Props = {
 function SettingsDropdown({ children, iconVariant, size, disableHoverPadding }: Props) {
   const [showDropdown, setShowDropdown] = useState(false);
 
-  const handleSettingsClick = useCallback(() => {
+  const handleSettingsClick = useCallback((event: React.MouseEvent<HTMLElement>) => {
+    event.preventDefault();
```

Comment:
Lmk if its not okay to add this event blocker here

Reply:
will address this!